src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person:
+  { name,
+    age,
+    sex,
+    isMarried,
+    partnerName },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age !== undefined
+      && <p className="Person__age">{`I am ${age}`}</p>}
```

Comment:
```suggestion
    {age && (
      <p className="Person__age">
        {`I am ${age}`}
      </p>
    )}
```
You can simplify the condition. Also, fix formatting